src/hgvs/dataproviders/uta.py

```diff
@@ -624,6 +624,11 @@ def _set_search_path(self, cur):
         cur.execute("set search_path = {self.url.schema},public;".format(self=self))
 
 
+class UTA_sqlite(UTABase):
```

Comment:
UTA_sqlite was removed long ago. I think the origin of this concern is uta.py:107, with a case for a sqlite scheme. We should remove that case rather than add an empty class.